cmip6_downscaling/disagg/derived_variables.py

```diff
@@ -1,147 +0,0 @@
-import numpy as np
```

Comment:
i could picture a world in which we want some of these (as well as the terraclimate parts) back at some point. but they aren't used now or in the near future (so deleting sounds great)

Reply:
totally. I can add these back if we think we're going to use them.